app/operations/organizations/create.rb

```diff
@@ -0,0 +1,21 @@
+module Organizations
+  class Create < Operation
+    string :name
+    string :display_name
+    string :primary_language
+
+    # Organization Contents Fields
+    string :title
+    string :description
+    string :introduction, default: ''
+
+    def execute
+      Organization.transaction do
+        organization = Organization.new owner: api_user.user, name: name, display_name: display_name, primary_language: primary_language
+        organization.organization_contents.build title: title, description: description, introduction: introduction, language: primary_language
+        organization.save!
+        organization
```

Comment:
I presume that this is due to the fact that `created_resource_response` takes the created object as an argument and grabs its updated_at, while `updated_resource_response` does not bother and just goes for the max: https://github.com/zooniverse/Panoptes/blob/master/app/controllers/concerns/json_api_responses.rb

Changing the way this works seems OOS for this PR.